GolfExample_TCAPI/index.html

```diff
@@ -156,6 +240,26 @@
 
             return newValue;
         }
+        
+        function saveBookmarking(callback){
```

Comment:
I'm going to need to find some time to do the publicity stuff once this PR is merged so it makes sense to pick up stuff like this at the same time. Can you keep a list of anything else as you go through? 

I'll also appreciate direction on a better way to achieve this. It looks like the horrible globals already existed and I'm extending that way of doing things rather than fixing it.